CHANGES.md

```diff
@@ -5,6 +5,8 @@
 
 * Added support for CoreOS 1688.4.0. (DCOS_OSS-2417)
 
+* Added check in custom installer for ftype=1 on Mesos and Docker work directories if using XFS
```

Comment:
This needs a JIRA in (brackets). I am not sure how much of the COPS ticket can be exposed, but it will helpful to have Open JIRA so that customers can look at the JIRA, and know more details.